source/particle/output/ascii.cc

```diff
@@ -106,19 +99,15 @@ namespace aspect
             output << "\n";
           }
 
-        file_index++;
+        this->increment_file_index ();
 
-        return output_file_prefix;
+        return "txt";
```

Comment:
This doesn't make much sense. The text is posted on the screen, so the real file name would be a useful thing to have. Or do you collate it later on from all output processors? If the latter, can you update the documentation of this function (in the base class) to explain what they are supposed to return? Or get the suffix from the base class via a new member function, given that the base class already knows the suffix?